src/l3-resources/common/props.ts

```diff
@@ -0,0 +1,319 @@
+import * as atlas from '@mongodbatlas-awscdk/cluster';
+import * as user from '@mongodbatlas-awscdk/database-user';
+import * as project from '@mongodbatlas-awscdk/project';
+import * as ipAccessList from '@mongodbatlas-awscdk/project-ip-access-list';
+
+/**
+ * @description
+ * @export
+ * @interface AtlasBasicProps
+ */
+export interface AtlasBasicProps {
```

Comment:
[q] Why not just call it `AtlasProps` - is there a reason to postpend basic?

Rationale: Basic makes it sound like low level of support, rather than the complete functionality.

Reply:
I believe it's so that the usage is easier to understand because it's used with `AtlasBasic` and `AtlasBasicPrivateEndpoint`, so maybe then we should consider changing all of that. lmk your thoughts